packages/pluggableWidgets/barcode-scanner-web/src/BarcodeScanner.xml

```diff
@@ -7,31 +7,24 @@
     <icon>iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAFgSURBVHgB7duxboMwEAbgP1GGCmZ4HtZOXYCZR4MVpk5deR9mULe2DKlUgrnaItJZ939TAibCJ3OWyfny9QOGXWEcAwDjbtsD4ziibVssy7J7QZZlqKoKRVFAk9D7fhgBwzA4f2Q1TRO6roM2ofd93WsoSdMU2szzHNTGOwesnS/LEto0TYM8z+HrJjXo+x4xWJ/tvbxU1/XhdZwGYRwDsD2QJMnv55Ckoo3Un4cA3LOp1mzvS+rPhatB4xgAGMcAwDhxLXD3+v755/vH20tU5134CMA4BgDGMQAwjgGAcQwAjGMAYBwDAOP4PgAnkdbfzz4fiiMAJ9H6xkfCJAjjmANwEs4CkeK/wzCOAYBxDACMc1aLr0VFGqvCfUn9cVaLa60K9yX157Ba/D8V2NpJ/WEShHGcBaQG23r7WPcMuXiPgFj3DLkcVou7xLpnyKta3CXWPUOsFnfgNAjjzAfgG6Bat42JKk75AAAAAElFTkSuQmCC</icon>
     <properties>
         <propertyGroup caption="General">
-            <propertyGroup caption="Data source">
+            <propertyGroup caption="General">
                 <property key="datasource" type="attribute" required="true">
-                    <caption>Barcode</caption>
-                    <description>The attribute that will receive the scanned barcode value.</description>
+                    <caption>Scanned result</caption>
+                    <description>The String attribute used to store the result of the scanned barcode.</description>
                     <attributeTypes>
                         <attributeType name="String"/>
                     </attributeTypes>
                 </property>
-            </propertyGroup>
-
-            <propertyGroup caption="Mask">
                 <property key="showMask" type="boolean" required="true" defaultValue="true">
                     <caption>Show barcode mask</caption>
-                    <description />
+                    <description>Apply a mask to camera view, as a specific target area for the barcode.</description>
                 </property>
             </propertyGroup>
 
             <propertyGroup caption="Events">
                 <property key="onDetect" type="action" required="false">
-                    <caption>On detect</caption>
-                    <description />
-                </property>
-                <property key="onClose" type="action" required="false">
-                    <caption>On close</caption>
-                    <description />
+                    <caption>On Detect Action</caption>
```

Comment:
We usually just capitalize the first sentence.

Reply:
Fixed 😄